src/test/pages/Issue.test.tsx

```diff
@@ -256,23 +253,21 @@ describe('issue form', () => {
 
     await waitFor(() => expect(mockIssueRequest).not.toHaveBeenCalled());
 
-    (window.bridge.btcRelay.getLatestBlockHeight as any).mockImplementation(() => MOCK_BTC_RELAY_HEIGHT);
-    (window.bridge.electrsAPI.getLatestBlockHeight as any).mockImplementation(() => MOCK_BITCOIN_HEIGHT);
+    mockBtcRelayGetLatestBlockHeight.mockImplementation(() => MOCK_BTC_RELAY_HEIGHT);
+    mockElectrsAPIGetLatestBlockHeight.mockImplementation(() => MOCK_BITCOIN_HEIGHT);
   });
 
   test('when the oracle is offline', async () => {
-    (window.bridge.oracle.getExchangeRate as any).mockImplementation(
+    mockOracleGetExchangeRate.mockImplementation(
       (currency: CurrencyExt) => new ExchangeRate(Bitcoin, currency, new Big(0))
     );
 
-    const { changeAmountToIssue, submitForm, tabPanel } = await renderIssueForm();
+    const { changeAmountToIssue, submitForm, errorElement } = await renderIssueForm();
 
     const inputAmount = 0.0001;
 
     await changeAmountToIssue(inputAmount.toString());
 
-    const errorElement = within(tabPanel).getByRole('alert');
-
     expect(errorElement.textContent).toMatchInlineSnapshot(
       `"You can't issue IBTC at the moment because oracle is offline."`
```

Comment:
Same as previous - there is probably couple more of these, I'm not going to comment on all of them.

Reply:
The same as above